rd_stats.h

```diff
@@ -543,6 +543,7 @@ struct stats_filesystem {
 	unsigned long long f_files		__attribute__ ((aligned (16)));
 	unsigned long long f_ffree		__attribute__ ((aligned (16)));
 	char 		   fs_name[MAX_FS_LEN]	__attribute__ ((aligned (16)));
+	char 		   mountp[MAX_FS_LEN]	__attribute__ ((aligned (16)));
```

Comment:
Attribute _((aligned (16))) not needed for field "mountp". It will be packed at the end of previous one ("fs_name").